h2o-core/src/main/java/hex/Model.java

```diff
@@ -449,6 +449,7 @@ public long getOrMakeRealSeed(){
     public String _offset_column;
     public String _fold_column;
     public String _treatment_column;
+    public String _id_column;
```

Comment:
In my opinion it should be only in the KNN algo and do not bother other models with ID. But I know its not a first stuff here..

Reply:
It has to be here because there are methods that require direct access to the column from the Model class. We don't bother other models with this new column. If we need it in some model, we can set it, otherwise it is not used.